migtests/tests/pg/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -1790,92 +1790,121 @@
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.mixed_data_types_table2.lseg_data (lseg)",
+					"ObjectName": "schema2.mixed_data_types_table2.lseg_data (lseg)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.mixed_data_types_table2.path_data (path)",
+					"ObjectName": "public.mixed_data_types_table2.lseg_data (lseg)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "schema2.mixed_data_types_table1.point_data (point)",
+					"ObjectName": "public.mixed_data_types_table1.lseg_data (lseg)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "schema2.mixed_data_types_table1.lseg_data (lseg)",
+					"ObjectName": "public.mixed_data_types_table1.point_data (point)",
 					"SqlStatement": ""
 				},
 				{
 					"ObjectName": "schema2.mixed_data_types_table1.box_data (box)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "schema2.mixed_data_types_table2.lseg_data (lseg)",
+					"ObjectName": "schema2.mixed_data_types_table1.point_data (point)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "schema2.mixed_data_types_table2.path_data (path)",
+					"ObjectName": "public.mixed_data_types_table1.box_data (box)",
 					"SqlStatement": ""
 				}
 			],
-			"DocsLink":"https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#unsupported-datatypes-by-voyager-during-live-migration",
+			"DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#unsupported-datatypes-by-voyager-during-live-migration",
 			"FeatureDescription": "There are some data types in the schema that are not supported by live migration of data. These columns will be excluded when exporting and importing data in live migration workflows."
 		},
 		{
 			"FeatureName": "Unsupported Data Types for Live Migration with Fall-forward/Fallback",
 			"Objects": [
-				{
-					"ObjectName": "public.combined_tbl.address (public.address_type)",
-					"SqlStatement": ""
-				},
 				{
 					"ObjectName": "public.documents.content_tsvector (tsvector)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.documents.title_tsvector (tsvector)",
+					"ObjectName": "public.orders.item (public.item_details)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.combined_tbl.arr_enum (public.enum_kind[])",
+					"ObjectName": "public.ts_query_table.query (tsquery)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.orders.item (public.item_details)",
+					"ObjectName": "public.combined_tbl.address (public.address_type)",
 					"SqlStatement": ""
 				},
 				{
 					"ObjectName": "public.products.item (public.item_details)",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "public.ts_query_table.query (tsquery)",
+					"ObjectName": "public.combined_tbl.arr_enum (public.enum_kind[])",
 					"SqlStatement": ""
 				},
 				{
-					"ObjectName": "schema2.orders.item (schema2.item_details)",
```

Comment:
Do you need all these changes other than the `EXTENSION` object or addition in the UQC section?

Reply:
It's mostly the shuffling change due to different runs. Can remove if required.

Reply:
Yeah, I think you should remove these changes and keep the ones required, it will keep changes minimal to the PR only.